src/Makefile

```diff
@@ -1,5 +1,4 @@
-OBJS = detection.o common.o battery.o logging.o encoder.o clock.o motor.o
-
+OBJS = $(filter-out main.o, (patsubst %.c,%.o,$(wildcard *.c))))
```

Comment:
```diff
diff --git a/src/Makefile b/src/Makefile
index de21979..2c70120 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -1,4 +1,4 @@
-OBJS = $(filter-out main.o, (patsubst %.c,%.o,$(wildcard *.c))))
+OBJS = $(filter-out main.o,$(patsubst %.c,%.o,$(wildcard *.c)))
 BINARY = main
```

Two small typos and it worked! :)
Could you fix and squash?

Reply:
Pushed to my Master branch with your fixes, but ´ git rebase -i master ´ doesn't work with any text editor i use :/, i'm really new to contributing with git really, i hope you don't get upset :stuck_out_tongue: